sklearnex/neighbors/_lof.py

```diff
@@ -97,6 +97,13 @@ def _onedal_fit(self, X, y, queue=None):
                 self.negative_outlier_factor_, 100.0 * self.contamination
             )
 
+        if sklearn_check_version("1.6"):

```

Comment:
Could you please reference where and why is it needed?

Reply:
scikit-learn#28773 which is already in planned inclusion in sklearn 1.6: https://scikit-learn.org/dev//whats_new/v1.6.html

Reply:
I guess Samir meant this sklearn issue/PR should be referenced in comment for better documented code. Anyway, I think it is good to start doing it in sklearnex for future explainability of code.